client/homebrew/editor/snippetbar/snippets/coverpage.gen.js

```diff
@@ -100,25 +100,25 @@ const subtitles = [
 
 module.exports = ()=>{
 	return `<style>
-  .phb#p1{ text-align:center; }
-  .phb#p1:after{ display:none; }
-  .phb#p2 { counter-reset:phb-page-numbers; }
-  .phb:nth-child(2n) .pageNumber { left: inherit !important; right: 2px !important; }
-  .phb:nth-child(2n+1) .pageNumber { right: inherit !important; left: 2px !important; }
-  .phb:nth-child(2n)::after { transform: scaleX(1); }
-  .phb:nth-child(2n+1)::after { transform: scaleX(-1); }
-  .phb:nth-child(2n) .footnote { left: inherit; text-align: right; }
-  .phb:nth-child(2n+1) .footnote { left: 80px; text-align: left; }
+  .page#p1{ text-align:center; counter-increment: none; }
+  .page#p1:after{ display:none; }
+  .page:nth-child(2n) .pageNumber { left: inherit !important; right: 2px !important; }
+  .page:nth-child(2n+1) .pageNumber { right: inherit !important; left: 2px !important; }
+  .page:nth-child(2n)::after { transform: scaleX(1); }
+  .page:nth-child(2n+1)::after { transform: scaleX(-1); }
+  .page:nth-child(2n) .footnote { left: inherit; text-align: right; }
+  .page:nth-child(2n+1) .footnote { left: 80px; text-align: left; }
 </style>
 
 <div style='margin-top:450px;'></div>
```

Comment:
Might as well change these divs to use the curly syntax.

`{{margin-top:450px}}`

Although curly divs / spans use display: inline-block which shifts the spacing a bit; 450px will no longer be correct. (In fact, looking at the current Div version on the live site, the top-margin on these spacings is going off the page in weird ways since `block` elements tend to clash with the column layout.